docs/guides/customize-your-scene.mdx

```diff
@@ -0,0 +1,758 @@
+  const scene = await viewer.scene();
+
+  // Hide all items where the `PART_NAME_KEY` includes a value of `PartName`
+  await scene.items((op) => [
+    op.where((q) => q.withMetadata('PartName', ['PART_NAME_KEY'])).hide(),
+  ]);
+
+  // Hide all items where the `PART_NAME_KEY` has exactly a value of `PartName`
+  await scene.items((op) => [
+    op.where((q) => q.withMetadata('PartName', ['PART_NAME_KEY'], true)).hide(),
+  ]);
+}
+```
+
+
+</details>
+
+#### `withPoint`
+
+- Parameters:
+  - `point: Point.Point` - the point on the image where an item to be operated on is located
+- Specifies that the operation should be performed on any item present at the provided `point` in the image. This query operates on the item found at that
+  `point` similar to using `withItemId` in combination with `raycaster.hitItems`, which can be useful if the additional metadata from the `raycaster.hitItems`
+  method is not needed to eliminate a network request.
+
+<details className="code">
+<summary>Example</summary>
+
+
+```js
+import { Point } from '@vertexvis/geometry';
+
+async function performOperation() {
+  const viewer = document.querySelector('vertex-viewer');
+  const scene = await viewer.scene();
+
+  // Select the item present at the [100, 100] coordinate of the image
+  await scene.items((op) => [
+    op.where((q) => q.withPoint(Point.create(100, 100))).select(),
+  ]);
+}
+```
+
+
+</details>
+
+#### `withSceneTreeRange`
+
+- Parameters:
+  - `range: SceneTreeRange` - the range within the tree to perform the operation on
+- Specifies that the operation should be performed on a range within the `<vertex-scene-tree>` component.
+
+<details className="code">
+<summary>Example</summary>
+
+
+```js
+async function performOperation() {
+  const viewer = document.querySelector('vertex-viewer');
+  const scene = await viewer.scene();
+
+  // Hide all items from the 2nd row to the 5th row of the scene-tree
+  await scene.items((op) => [
+    op
+      .where((q) =>
+        q.withSceneTreeRange({
+          start: 2,
+          end: 5,
+        })
+      )
+      .hide(),
+  ]);
+}
+```
+
+
+</details>
+
+#### `withVolumeIntersection`
+
+- Parameters:
+  - `rectangle: Rectangle.Rectangle` - the 2D bounds within the image
+  - `exclusive?: boolean` - whether this is exclusive (only items fully contained) or inclusive (items fully contained + intersecting)
+- Specifies that the operation should be performed on items within the specified `rectangle` boundary within the Viewer. The `exclusive` flag here determines
+  whether items that intersect with the `rectangle`, but are not contained should be included in the result.
+
+<details className="code">
+<summary>Example</summary>
+
+
+```js
+import { Rectangle } from '@vertexvis/geometry';
+
+async function performOperation() {
+  const viewer = document.querySelector('vertex-viewer');
+  const scene = await viewer.scene();
+
+  // Select all items within the specified 100x100 region of the image
```

Comment:
Yeah, good thought. I'll add a comment to explain where this is positioned! I'd be hesitant to use the screen center, since this is relative to the viewer's viewport, but maybe I can add an explanation of that as well